packages/manager/modules/pci/src/components/users/download-rclone/translations/Messages_fr_FR.json

```diff
@@ -9,5 +9,6 @@
   "pci_projects_project_users_download-rclone_region_help": "Pour des raisons de sécurité le fichier de configuration ne contient pas votre mot de passe. Il est donc nécessaire d'éditer le fichier de configuration pour y ajouter manuellement votre mot de passe.",
   "pci_projects_project_users_download-rclone_success_message": "Le téléchargement du fichier de configuration rclone a commencé, si ce n'est pas le cas ",
   "pci_projects_project_users_download-rclone_success_message_link": "cliquez-ici",
-  "pci_projects_project_users_download-rclone_error_rclone": "Une erreur est survenue lors du téléchargement du fichier de configuration rclone : {{ message }}."
+  "pci_projects_project_users_download-rclone_error_rclone": "Une erreur est survenue lors du téléchargement du fichier de configuration rclone : {{ message }}.",
+  "pci_projects_project_users_download-rclone_file_type_label": "Type File"
```

Comment:
`File` shouldn't be translated?

Reply:
Hello Jeremy, I have already asked same question to the QA. I am waiting for the reply.

Reply:
Updated the file with right value.